packages/graphback-runtime-mongodb/src/MongoDBDataProvider.ts

```diff
@@ -20,6 +21,33 @@ export class MongoDBDataProvider<Type = any, GraphbackContext = any> implements
     this.db = db;
     this.tableMap = buildModelTableMap(baseType);
```

Comment:
I am very curious about this field and what it is being used for besides idField

Reply:
This is being used for id field mapping and the table name => GraphQLObjectType name mapping.

The eventual aim was to also do a GraphQL field ==> column name mapping and this is what `fieldMap` is for. This is partially implemented but we never actually used it in the database providers, because it is a lower priority for us right now.